charts/prometheus-druid-exporter/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: v1
+appVersion: "v0.8.0"
+description: Druid exporter to monitor druid metrics with Prometheus
+engine: gotpl
+name: prometheus-druid-exporter
+sources:
+  - https://github.com/opstree/druid-exporter
+version: 0.8.0
+home: https://github.com/opstree/druid-exporter
+maintainers:
+- email: abhishekbhardwaj510@gmail.com
+  name: iamabhishek-dubey
+- email: sandeep@opstree.com
+  name: sandy724
+keywords:
+- kubernetes
+- apache
+- druid
+- exporter
+- prometheus
+- grafana
+icon: https://github.com/opstree/druid-exporter/blob/master/static/druid-exporter-logo.svg
+annotations:
+  "artifacthub.io/operator": "true"
+  "artifacthub.io/links": |
+    - name: Chart Source
```

Comment:
In order for this annotation to display in artifact hub the link must be in the `sources` list (it matches by URL).

But also see this comment I made just now about simplifying what I would have done earlier in retrospect https://github.com/prometheus-community/helm-charts/pull/114#issuecomment-695335157

Reply:
To clarify, in order for the Artifact Hub links annotation to overwrite the link name, the URL must already exist in the `sources` list. But now the sources list has been removed.

In fact, I may even suggest adding a direct link to the chart directory within this github repo as the source. It would be `https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-druid-exporter`

Reply:
Looks like you've got it